src/_assets/stylesheets/components/_layout.scss

```diff
@@ -18,6 +18,29 @@
   }
 }
 
+.homepage-hero {
+  margin-bottom: $site-margins;
+  padding: 0;
+
+  @include media($nav-width) {
+    @include padding(null $site-margins);
+  }
+
+  h2 {
+    @include padding(6rem $site-margins-mobile);
+    background-color: $color-primary;
+    color: #fff;
+    font-family: $font-sans;
+    font-weight: $font-normal;
+    text-align: center;
+
+    @include media($nav-width) {
+      @include padding(6rem null);
+      font-size: $h1-font-size;
+    }
+  }
+}
```

Comment:
I'm 🤷‍♂️ on where to best organize styles like these and those in `src/_assets/stylesheets/components/_section.scss` and `src/_assets/stylesheets/elements/_typography.scss`. The general design goal is to mimic how the US Web Design Standards' file organization, but that only gets us so far. May reconsider this organization in a future PR.